Resources/Audio/Animals/license.txt

```diff
@@ -47,3 +47,9 @@ The following sounds were used from youtube:
     lizard_happy.ogg: audio created by youtube user "Nagaty Studio" (https://youtu.be/I7CX0AS8RNI) licensed under CC by 3.0.
 
     bear.ogg: audio is recorded by "Nagaty Studio" (https://www.youtube.com/watch?v=pz6eZbESlU8) licensed under CC by 3.0. The original audio was reverbed.
+
+The following sounds were used from pixabay:
+
+    cerberus.ogg: audio is created by pixabay deleted user 3424813 (https://pixabay.com/sound-effects/dog-snarl-self-made-105738/) licensed under CC0.
+
+    bite_demon.ogg: audio is created by pixabay user nick121087 (https://pixabay.com/sound-effects/demonic-woman-scream-6333/) licensed under CC0. A sample was taken and it has been mixed with bite.ogg
```

Comment:
where does the website lists the licenses of these

Reply:
damn, I'll just find something in the files